src/add-to-list.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-cycle */
+
+import displayTasks from './index.js';
+
+const addTask = document.getElementById('task');
+const form = document.getElementById('my-form');
+
+const addTaskList = (taskList) => {
+  const description = addTask.value;
+  const completed = false;
+  if (description === '') {
+    addTask.setAttribute('required', '');
+    addTask.style.border = 'thin solid red';
+  } else {
+    const id = taskList.length + 1;
+    const task = { description, completed, id };
+    taskList.push(task);
+    addTask.style.border = 'thin solid black';
+    localStorage.setItem('taskList', JSON.stringify(taskList));
+    displayTasks();
+  }
+  form.reset();
+};
```

Comment:
- [OPTIONAL] It would be nice to prevent adding duplicate tasks. Don't you think?